src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee

```diff
@@ -23,10 +23,16 @@
         offset = (page  - 1) * nbItems
         fetchOrganizations(nbItems, offset)
 
+    listingFilters = (term) ->
+      term[ 'account_frozen' ] = !scope.hideFrozen if scope.hideFrozen
+      term[ 'demo_account' ] = !scope.hideDemo if scope.hideDemo
```

Comment:
I think this would be more readable:
```ruby
term[ 'account_frozen' ] = false if scope.hideFrozen
term[ 'demo_account' ] = false if scope.hideDemo
```